packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -44,9 +44,7 @@ const SettingsSidebar = ( { sidebarName } ) => (
 				</>
 			) }
 			{ sidebarName === 'edit-post/block' && (
-				<PanelBody className="edit-post-settings-sidebar__panel-block">
```

Comment:
For better or for worse, I think the idea originally is that each `Panel`  should contain `PanelHeader` or `PanelBody` as its children and not random divs. This is very old though so I'm not sure how true is that these days but it does speak to the fact that maybe our Panel*  components are not well thought if we hack them this way.

Reply:
In the document sidebar we render a Panel and then multiple PanelBody's, we also render a Panel and then we will have multiple panel body.
As when the blocks add things to the inspector they add PanelBodys without a panel above:
```
				<InspectorControls>
					<PanelBody title={ __( 'Text Settings' ) } className="blocks-font-size">
```

The only difference between block inspector and document is that between the panel and panelbody we have some div's but I think it is something expected otherwise if the intention was to not allow that, panel could have just received a bodies prop with an array of things to use as the body and it could not even expose the panel body.
I think previously we were even in a worse position as it looks like we had panel body and then other panel bodies inside. Now we have panel some divs for styling and then panel bodies.